src/lib/builtins/boolean.rs

```diff
@@ -91,8 +91,12 @@ pub fn this_boolean_value(value: &Value) -> Value {
 mod tests {
     use super::*;
     use crate::exec::Executor;
+<<<<<<< HEAD:src/lib/builtins/boolean.rs
```

Comment:
This syntax .`<<<<<<` is from a rebase, its git syntax not rust.
It should  not be there and its causing the checks/builds to fail